frontend/micro-ui/web/micro-ui-internals/packages/modules/core/src/pages/citizen/Login/SelectMobileNumber.js

```
@@ -1,87 +1,307 @@
-  }
+    return (
+      <span>
+        {isCCFEnabled?.checkBoxLabels?.map((data, index) => (
+          <span key={index}>
+            {data?.linkPrefix && <span>{t(`${data?.linkPrefix}_`)}</span>}
+            {data?.link && (
+              <span
+                id={data?.linkId}
+                onClick={(e) => onLinkClick(e)}
+                style={{ color: "#F47738", cursor: "pointer" }}
+              >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)